src/app/rundown-view/components/rundown/rundown.component.ts

```diff
@@ -49,14 +53,16 @@ export class RundownComponent implements OnInit, OnDestroy, OnChanges {
       .then(rundownTimingContextObservable => rundownTimingContextObservable.subscribe(this.onRundownTimingContextChanged.bind(this)))
       .then(rundownTimingContextSubscription => (this.rundownTimingContextSubscription = rundownTimingContextSubscription))
       .catch(error => this.logger.data(error).error('Failed subscribing to rundown timing context changes.'))
-    this.subscribeForEventObserver()
+    this.subscribeToEventObserver()
   }
 
-  private subscribeForEventObserver(): void {
+  private subscribeToEventObserver(): void {
     this.rundownEventSubscriptions.push(this.rundownEventObserver.subscribeToRundownAutoNext(this.setAutoNextStartedToTrue.bind(this)))
     this.rundownEventSubscriptions.push(this.rundownEventObserver.subscribeToRundownSetNext(this.setAutoNextStartedToFalse.bind(this)))
     this.rundownEventSubscriptions.push(this.rundownEventObserver.subscribeToRundownReset(this.setAutoNextStartedToFalse.bind(this)))
     this.rundownEventSubscriptions.push(this.rundownEventObserver.subscribeToRundownDeactivation(this.setAutoNextStartedToFalse.bind(this)))
+    this.rundownEventSubscriptions.push(this.rundownEventObserver.subscribeToRundownTake(this.scrollViewToSegment.bind(this)))
+    this.rundownEventSubscriptions.push(this.rundownEventObserver.subscribeToRundownSetNext(this.scrollViewToSegment.bind(this)))
```

Comment:
This component already subscribes to `RundownSetNext` on line 61. No need to make two subscriptions for the same thing.
A potential way of doing it:
```
this.rundownEventSubscriptions.push(this.rundownEventObserver.subscribeToRundownSetNext(event => {
  this.setAutoNextStartedToFalse(event)
  this.scrollViewToSegment(event)
}))
```

Reply:
Changed to the suggestion, good point!